overlord/devicestate/handlers_install.go

```diff
@@ -1080,7 +1082,7 @@ func (m *DeviceManager) doInstallFinish(t *state.Task, _ *tomb.Tomb) error {
 
 		logger.Debugf("copying label %q to seed partition", systemAndSnaps.Label)
 		if err := copier.Copy(seedMntDir, seed.CopyOptions{
-			Label:              systemAndSnaps.Label,
+			Label:              systemLabel,
```

Comment:
I think it is probably better to still use systemAndSnaps.Label, note that we are using this object to get any seed information.